packages/cli/src/cmd/serve.js

```diff
@@ -63,13 +75,26 @@ function serve(userSpecifiedRoot, options) {
     logLevel: 0,
     root: outputFolder,
     port: options.port || 8080,
+    host: options.address || '127.0.0.1',
     middleware: [],
     mount: [],
   };
 
   site
     .readSiteConfig()
     .then(async (config) => {
+      if (serverConfig.host === '0.0.0.0') {
+        const response = await questionAsync(
+          'WARNING: Using the address \'0.0.0.0\' may pose vulnerabilities and is generally discouraged.'
+          + 'Proceed with caution? [y/N] ');
```

Comment:
Perhaps the vulnerabilities could be made more explicit, maybe something along the lines of ... could expose your server to the internet, which may pose security risks. .... As I feel that if the user is not aware of what vulnerabilities there are, they may choose to ignore the warning without fully understanding the situation. The rest of the message seems good!

Reply:
Yup, I believe that would be better, thanks!

Reply:
Thanks for making the change the latest warning message looks good to me!